guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-manually.adoc

```diff
@@ -0,0 +1,57 @@
+[id="deploying-the-ca-certificate-on-a-host-manually"]
+= Deploying the CA certificate on a host manually
+
+You can deploy the CA certificate on the host manually by rendering a public provisioning template, which will provide the certificate to your host.
+Redeploy the CA certificate when you change it on {ProjectServer}.
+
+.Prerequisites
+* You have root access on your host.
+
+.Procedure
+. Log in to you host by using SSH.
+. Download the certificate on your host:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+curl -o "foreman_ca_cert.crt" http://_{foreman-example-com}_/unattended/public/foreman_raw_ca
+----
++
+Replace _{foreman-example-com}_ with the hostname of your {ProjectServer}.
+. Copy the certificate to the truststore:
+ifdef::client-content-dnf[]
+* On {EL}:
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+cp foreman_ca_cert.crt /etc/pki/ca-trust/source/anchors
+----
+endif::[]
+ifdef::client-content-apt[]
```

Comment:
@maximiliankolb Are these attributes used correctly?

Reply:
Yes, based on the rendered docs: LGTM.